kubectl-rook-ceph.sh

```diff
@@ -263,6 +265,87 @@ function run_purge_osd() {
     rook ceph osd remove --osd-ids=$1 --force-osd-removal=$force_removal"
 }
 
+####################################################################################################
+# 'kubectl rook-ceph debug commands
+####################################################################################################
+
+function update_deployment_spec_command(){
+  deployment_spec=$1
+  main_command="[\"sleep\",\"infinity\"]"
+  deployment_spec=$(jq .template.spec.containers[0].command="$main_command" <<< $deployment_spec)
+  deployment_spec=$(jq .template.spec.initContainers[1].command="$main_command" <<< $deployment_spec)
+  deployment_spec=$(jq .template.spec.initContainers[0].command="$main_command" <<< $deployment_spec)
+}  
+
+function update_deployment_spec_image(){
+  deployment_spec=$1
+  alternalte_image=$2
+  deployment_spec=$(jq .template.spec.containers[0].env[0].value="$alternalte_image" <<< $deployment_spec)
```

Comment:
Are we sure??

Reply:
> Are we sure about the env, because I think it is needed for container env??

Reply:
From looking at the Ceph code, I believe the var is just used to get the name of the image from debug logs. But since we are in a manual debug scenario, it shouldn't matter.